python/tests/unit/test_import_hygiene.py

```
@@ -0,0 +1,65 @@
+import subprocess
+import sys
+from pathlib import Path
+
+_PYTHON_ROOT = Path(__file__).resolve().parents[2]
+
+
+def _run_isolated_import_check(
+    script: str,
+) -> subprocess.CompletedProcess[str]:
+    """Run import check in a subprocess (does not pollute pytest sys.modules)."""
+    return subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)